src/Commands/Multidev/MergeFromDevCommand.php

```diff
@@ -15,20 +15,20 @@ class MergeFromDevCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Merge the changes from the dev environment into a multidev environment
+     * Merges code commits from the Dev environment into a Multidev environment.
      *
      * @authorize
      *
      * @command multidev:merge-from-dev
      * @aliases env:merge-from-dev
      *
-     * @param string $site_env Site & environment to merge changes from dev into, in the form `site-name.env`.
-     * @option boolean $updatedb True to update the DB along with this merge
+     * @param string $site_env Site & Multidev environment in the form `site-name.env`
+     * @option boolean $updatedb Run update.php afterwards
      *
      * @usage terminus multidev:merge-from-dev <site>.<env>
```

Comment:
Some of these commands use `<site>.<multidev>` and others `<site>.<env>` for multidev environments. They should be consistent.